packages/jest-cli/src/TestSequencer.js

```diff
@@ -59,6 +72,41 @@ export default class TestSequencer {
     return cache;
   }
 
+  _fileSizeRecurseDependencies(test: Test, sizes: Map<Path, number>): number {
+    const {resolver, hasteFS, config} = test.context;
+
+    const fileSize = (path): number => {
```

Comment:
by passing `sizes` to this fn, you could extract it outside of `_fileSizeRecurseDependencies`, so it doesn't have to be recreated multiple times

Reply:
Note that it also uses `resolver` and `hasteFS`, so `fileSize` would need the whole test instead of just the path as its argument as well. Not sure if that's worth it?

Reply:
Yup, so it can be a class method, right?

Reply:
Converted it to a class method, although IMHO it's less readable now.